sidecar.Dockerfile

```diff
@@ -15,6 +15,27 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 RUN echo "allow_root" >> /etc/fuse.conf
 
+## BEGIN tini
+
+ENV TINI_VERSION v0.18.0
```

Comment:
Do we really need to install tini? Recent docker includes a `docker  run --init` [option](https://docs.docker.com/engine/reference/commandline/container_run/) . As per the author of tini himself (https://github.com/krallin/tini#using-tini) we should no more have to install this. To be tested, of course...

Reply:
the tini use is a copy 'n paste job throughout.

i can consider it technical debt, yet i'm pressed to consider it part of the scope of this patch.